src/common_modules/vsts-cli-common/vsts/cli/common/_credentials.py

```diff
@@ -55,7 +55,6 @@ def clear_credential(team_instance):
     try:
         keyring.delete_password(key, _USERNAME)
     except keyring.errors.PasswordDeleteError as ex:
-        logger.exception(ex)
```

Comment:
I removed this one since it's almost certainly user error. There were other logger.exception calls in common code that I didn't remove since it wasn't obvious if they were user errors or not. There are also logger.exception calls in specific product areas that I didn't touch (but can if you'd like).